recipe/meta.yaml

```
@@ -28,32 +28,31 @@ requirements:
   build:
     - python                                 # [build_platform != target_platform]
```

worth diffing against the upstream recipe in the text editor of your choice.

One departure from the conda-forge documentation is that we are stricter about build/host separation. We always need python in the build environment because we're going to enforce that the CMake configuration used the ``BUILD_PREFIX`` during conda recipe builds. This is what allows us to move python out of the runtime dependencies in packages like vector tools, but retain the python/setuptools_scm dependence for versioning in the build environment.

```suggestion
    - python
```

---

there are more changes required in the build section, too. See https://github.com/UCBoulder/tardigrade_error_tools/blob/dev/recipe/meta.yaml#L27-L44. Almost everything is moved into build because most of these packages are only required at build time and are not required on the host/target platform.

---

Cool, I wasn't sure if I should make this change here or if it was specific to our needs.

---

I tried to mark in-repo specific changes in vector tools. I wasn't quite as complete in error tools.

If it would help, we can go back into the in-repo recipe and comment on lines that are expected to differ from the conda-forge recipe.

---

https://github.com/UCBoulder/tardigrade_error_tools/pulls